docs/Usage/RemoteKernelConnection.md

```diff
@@ -1,12 +1,12 @@
 # Remote kernels via kernel gateways
 
-In addition to managing local kernels and connecting to them over ZeroMQ, Hydrogen is also able to connect to Jupyter Notebook (or Jupyter Kernel Gateway) servers. This is most useful for running code remotely (e.g. in the cloud).
+In addition to managing local kernels and connecting to them over ZeroMQ, Hydrogen is also able to connect to Jupyter Notebook (or Jupyter Kernel Gateway) servers. This is most useful for running code remotely (e.g. in the cloud), or in a Docker container running locally.
 
 To connect to a server, you must first add the connection information to the Hydrogen `gateways` setting. An example settings entry might be:
 
 ```json
 [{
-  "name": "Remote notebook",
+  "name": "Remote kernel",
```

Comment:
The name actually refers to the server (not to a particular kernel), so better to say something like "Remote server".

Reply:
👍 for "Remote server"